simple_test.sh

```diff
@@ -1,48 +1,81 @@
 #!/bin/sh
 
+# This test script automates our distributed work system. When running this script, please run it as follows:
+# ./simple_test [hostname] [# of workers] [test filename]
```

Comment:
This type of thing should be printed in a help message

Reply:
Don't include test directory when running. Just filename. It assumes all of our tests are coming from the test directory - this makes sense with how our workflow is setup. Zach and I talked about adding some error messages, which I drafted up yesterday. I'll commit these changes later today.

Reply:
It still didn't work when I removed `test/`

Reply:
Also @Bronstrom  I don't think assuming it is in test/ is good. We need it to not be purely for testing. It will be a general script to just run. I rarely use scripts in /test.